ChiModules/CFEMDiffusion/lua/ds_lua_utils.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CFEM_DIFFUSION_LUA_UTILS_H
+#define CFEM_DIFFUSION_LUA_UTILS_H
+
+#include"ChiLua/chi_lua.h"
+#include "../cfem_diffusion_solver.h"
+
+int chiCFEMDiffusionSolverCreate(lua_State *L);
+int chiCFEMDiffusionSetBCProperty(lua_State *L);
+
+
+namespace cfem_diffusion
+{
+  namespace cfem_diffusion_lua_utils
+  {
```

Comment:
No Action. I removed the GetSolverHandle business since it was not used and we now have better tech to do it, e.g., `GetStackItem`.

Reply:
So will you update LSBKeigen solver? I am still shaky on the LUA wrappers so I am not sure what you meant